voltixApp/VoltixApp/Utils/Endpoint.swift

```diff
@@ -38,45 +38,73 @@ class Endpoint {
         "http://45.76.120.223/blockchair/\(coinName)/dashboards/address/\(address)"
     }
     
-    static func ethereumLabelTxHash(_ value: String) -> String {
-        "https://etherscan.io/tx/\(value)"
-    }
-    
-    static func fetchUnspentOutputs(_ value: String) -> String {
+	static func ethereumLabelTxHash(_ value: String) -> String {
+		"https://etherscan.io/tx/\(value)"
+	}
+	
+	static func fetchUnspentOutputs(_ value: String) -> String {
         "http://45.76.120.223/blockcypher/v1/btc/main/addrs/\(value)?unspentOnly=true"
-    }
+	}
     
     static func fetchLitecoinUnspentOutputs(_ userAddress: String) -> String {
         "https://litecoinspace.org/api/address/\(userAddress)/utxo"
     }
     
-    static func fetchCryptoPrices(coin: String, fiat: String) -> String {
-        "https://api.coingecko.com/api/v3/simple/price?ids=\(coin)&vs_currencies=\(fiat)"
-    }
-    
-    static func fetchBitcoinTransactions(_ userAddress: String) -> String {
-        "https://mempool.space/api/address/\(userAddress)/txs"
-    }
-    
+	static func fetchCryptoPrices(coin: String, fiat: String) -> String {
+		"https://api.coingecko.com/api/v3/simple/price?ids=\(coin)&vs_currencies=\(fiat)"
+	}
+	
+	static func fetchBitcoinTransactions(_ userAddress: String) -> String {
+		"https://mempool.space/api/address/\(userAddress)/txs"
+	}
+	    
     static func fetchLitecoinTransactions(_ userAddress: String) -> String {
         "https://litecoinspace.org/api/address/\(userAddress)/txs"
     }
-    
-    static func getEthInfo(_ address: String) -> String {
-        "https://api.ethplorer.io/getAddressInfo/\(address)?apiKey=freekey"
-    }
-    
-    static func broadcastEtherscanTransaction(hex: String) -> String {
-        "http://45.76.120.223/etherscan/api?module=proxy&action=eth_sendRawTransaction&hex=\(hex)"
-    }
-    
-    static func fetchEtherscanTransactions(address: String) -> String {
-        "http://45.76.120.223/etherscan/api?module=account&action=txlist&address=\(address)&startblock=0&endblock=99999999&sort=asc"
-    }
-    
-    static func fetchERC20Transactions(address: String, contractAddress: String) -> String {
-        "http://45.76.120.223/etherscan/api?module=account&action=tokentx&contractaddress=\(contractAddress)&address=\(address)&startblock=0&endblock=99999999&sort=asc"
-    }
+	    
+	static func getEthInfo(_ address: String) -> String {
+		"https://api.ethplorer.io/getAddressInfo/\(address)?apiKey=EK-qHhpP-qzAvwsm-WSsW3"
```

Comment:
Could you remove this , if we don't use it anymroe

Reply:
Done